src/trace.ml

```diff
@@ -272,23 +272,46 @@ module Make_commands (Backend : Backend_intf.S) = struct
   let record_dir_flag mode =
     let open Command.Param in
     flag
-      "-record-dir"
+      "-temporary-data-directory"
```

Comment:
Don't think `temporary` is quite the right term here, given this directory explicitly doesn't get cleaned up.